client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -21,51 +20,44 @@ import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
 import SectionHeader from 'components/section-header';
 import { getSelectedSiteSlug } from 'state/ui/selectors';
 
-const ZoneCreator = ( { siteSlug, translate } ) => {
-	const handleGoBack = () => {
-		page( `/extensions/zoninator/${ siteSlug }` );
-	};
+const form = 'extensions.zoninator.newZone';
 
-	return (
-		<div>
-			<HeaderCake onClick={ handleGoBack }>
-				{ translate( 'Add a zone' ) }
-			</HeaderCake>
+const ZoneCreator = ( { siteSlug, translate } ) => (
+	<div>
+		<HeaderCake backHref={ `/extensions/zoninator/${ siteSlug }` } onClick={ noop }>
```

Comment:
Is the `onClick={ noop }` needed? I know it's required but I'm just wondering why @ockham got away without specifying it in his PR?